docs/data/introduction/licensing/licensing.md

```diff
@@ -213,6 +213,11 @@ For example, if you purchase a one-year license today, you will be able to updat
 Those versions will always be available for use in a deployed application,
 however you'll be required to renew your license if you need to continue development with a version released after twelve months.
 
+#### Out of scope license key
+
+This error indicates that the component you are trying to use is not included in your license.
+This can happen if you try to use `DataGridPremium` with a pro license.
```

Comment:
To match the capitalization used in https://mui.com/x/introduction/licensing/#upgrading-from-pro-to-premium?

```suggestion
This can happen if you try to use `DataGridPremium` with a Pro license.
```